tests/moduletool/test_python_dependencies.py

```diff
@@ -54,3 +55,29 @@ def test_collect_python_requirements(tmpdir):
     reqs = project.collect_python_requirements()
     expected_reqs = ["iplib@git+https://github.com/bartv/python3-iplib", "pytest>=1.5", "iplib>=0.0.1", "dummy-yummy"]
     assert sorted(reqs) == sorted(expected_reqs)
+
+
+def test_requirements_from_source_info(tmpdir):
+    common.makeproject(tmpdir, "test-project", deps=[("mod1", "")], imports=["mod1"])
+    project_dir = os.path.join(tmpdir, "test-project")
+    libs_dir = os.path.join(project_dir, "libs")
+    common.makemodule(libs_dir, "mod1", project=False)
+    mod1 = os.path.join(libs_dir, "mod1")
+    mod1_req_txt = """# I'm a comment
+    pytest\
+    >=\
+    1.5
+
+
+    iplib>=0.0.1
+
+        """
+    common.add_file(mod1, "requirements.txt", mod1_req_txt, msg="initial commit")
+    project = Project(project_dir)
+    project.load_module("mod1")
+    Project.set(project)
+    requirements = SourceInfo(mod1, "inmanta_plugins.mod1").requires
+    assert "# I'm a comment" not in requirements
```

Comment:
Let's explicitly verify that the list of requirements being returned, is correct.